examples/import/1524910028_examples_down.sql

```diff
@@ -0,0 +1,6 @@
+/*
+	Code generated by protoc-gen-orm. DO NOT EDIT.
+	source: Example
+*/
+
+DROP TABLE IF EXISTS examples;
```

Comment:
For the migration files my idea was to take a flag input from user if he wants this file to be generated. something like `--migrations=.`, user provides a directory where to generate the files instead of the same proto directory.